src/lib/Components/CustomModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { theme } from "lib/Components/Bidding/Elements/Theme"
+import React from "react"
+import { Modal, TouchableWithoutFeedback, View, ViewProperties } from "react-native"
+import styled from "styled-components/native"
+import { Sans14, SansMedium14 } from "./Bidding/Elements/Typography"
+import { SecondaryOutlineButton } from "./Buttons"
+
+interface CustomModalProps extends ViewProperties {
+  headerText: string
+  detailText: string
+  visible?: boolean
+  closeModal?: () => void
+}
+
+const ModalBackgroundView = styled.View`
+  background-color: #00000099;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+const ModalInnerView = styled.View`
+  width: 300;
+  background-color: white;
+  padding: 20px;
+  opacity: 1;
+  border-radius: 2px;
+`
+
+export class CustomModal extends React.Component<CustomModalProps, any> {
```

Comment:
I think we can call this component just `Modal`?

cc @orta @ashfurrow

Reply:
Ohh, I wasn't sure if we wanted to do that since there's also a react-native one. But happy to change!

Reply:
Yeah I'm fine with just `Modal`, if someone is importing both in another file they can rename it using `import { Modal as CustomModal } from "./Modal"` or similar 👍